botocore/utils.py

```diff
@@ -48,6 +48,20 @@
 RETRYABLE_HTTP_ERRORS = (requests.Timeout, requests.ConnectionError)
 S3_ACCELERATE_WHITELIST = ['dualstack']
 
+CONTROLS = {
```

Comment:
I think I would prefer this lives in botocore/docs/utils.py because it is really only used in the documentation and the `botocore/docs/utils.py` is more internal than `botocore/utils.py` module